README.md

```
@@ -1,15 +1,156 @@
-# 🧠 Deepnote
+![Deepnote cover image](deepnote-cover-image.png)
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)